_docs/tutorial/04-testing-in-production.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Testing in production
+overview: Overview
+
+order: 04
+
+layout: docs
+type: markdown
+---
+{% include home.html %}\n# Testing a microservice in "production"
+
+Let's perform some testing of our microservice . This learning module exemplifies that you can test your microservices in production!
+
+1. Let's send some requests to our microservice from inside the cluster, we will use a dummy pod, `sleep`.
+   ```
+   kubectl apply -f ../../istio-*/samples/sleep/sleep.yaml
+   ```
+
+   Once the `sleep` pod is ready, we can issue HTTP requests from it to our service and test it
+   ```
+   kubectl exec -it $(kubectl get pod -l app=sleep -o jsonpath='{.items[0].metadata.name}') bash
+   curl http://ratings:9080/ratings/7
+   ```
+1. Let's do some [chaos testing](http://www.boyter.org/2016/07/chaos-testing-engineering/) in production and see how our application reacts. After each chaos operation, access your `http://<your host>/productpage` and see if anything  changed. Also check the pods status with `kubectl get pods`.
+   1. Let's kill the _details_ service in one pod.
```

Comment:
Here and elsewhere: kill -> terminate

Reply:
![image](https://user-images.githubusercontent.com/1923135/36991270-04a462fc-205c-11e8-8a70-2016701987db.png)